api/src/integrations/integration-datasource-enum.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Supported NRPTI datasources.
+ */
+const INTEGRATION_DATASOURCE = Object.freeze({
+  epic: {
+    dataSourceLabel: 'epic',
+    getDataSource: (...args) => {
+      return new (require('./epic/epic-datasource'))(...args);
+    }
+  }
+});
+
+module.exports = INTEGRATION_DATASOURCE;
```

Comment:
The idea is that when we get more datasources, they all live here.  Currently we only have 1.

Also similar to the schema enum, this relates the swagger datasource type string to the display string, since they might not match (or might contain spaces, etc).

And also, its an alternative to big switch statements, because now you can just do:

```
const dataSourceType = INTEGRATION_DATASOURCE[swagger.stuff.datasourceType]
```

Reply:
If this just replaces a switch statement, isn't this more complicated and harder to maintain then a single switch statement kept within the import-task file?

Reply:
You could technically do everything this is doing with a switch, since the switch could return the same object this does.  So in that regard its probably just a preference. Though maybe this is marginally faster, since you can jump straight to the datasource you want rather than iterating a switch?

In either case, I would argue that the logic should still be in a util file, like this one.  
That way its re-usable and no code duplication is needed to call this elsewhere, and easily testable because there is no other code to have to mock out for these tests.   It also keeps the test code for the main calling classes smaller because they have one less thing to worry about.